physics/biot_savart_spacepy.py

```diff
@@ -1,15 +1,44 @@
 """
 typical output:
 
-opening: /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/3d__var_3_e20031120-174600-259.out
----------
-dBmhd = 1011.2855377944559
-fractional error = -0.20493277341394997
------------------------
-opening: /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/3d__var_3_e20031120-174500-000.out
----------
-dBmhd = 1004.0975310504954
-fractional error = -0.24662938647340985
+Reading /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/ls-1.txt
+Read /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/ls-1.txt
+(117, 8)
+opening: /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/3d__var_3_e20031120-070000-000.out
+---
+Event: 2003,11,20,7,0;55.00,241.00
```

Comment:
Print out info about how the calculation was made. `dBmhd` is `dB` in SWMF output, etc. When reading the log file, I should be able to understand what method you were using to compute dB.

Reply:
You can avoid having to copy/paste "typical output" by running the following on the command line

```
python biot_savart_spacepy.py > biot_savart_spacepy.log
```

Then commit the log file and reference it in your comment.

Reply:
when trying this, the resulting biot_savart_spacepy.log appears to be a blank document.

Reply:
This is somewhat unexpected.

`python biot_savart_spacepy.py` should result in text being shown in the terminal. When you use `>`, it sends everything you see on the terminal to the file that follows.

Look into `stdout` and `stderr` to learn more about the `>` and `|` shell operators.

Reply:
ok now I found it. It looks like they make a blank copy and put it in a different folder